# Chapter 03
## Processes and Process Variables

- ** Process ** - any operation or series of operations by which a particular objective is accomplished
- ** input or feed ** - material that enters a process
- ** output or product ** - material that leaves a process
- ** process unit ** - a single step of a process
- ** process streams ** - set inputs and outputs of a single process unit

- ** Design ** - formulation of a process flowsheet (layout), specification of process units (i. e. reactors, separation equipment, heat exchangers, etc.)
- ** Operation ** - day-to-day running of the proces
- ** Debottle-necking ** - troubleshooting and experimenting to find how to increase production rate greater than that which can be accomodated by current equipment at current operating conditions

## 3.1 Mass and Volume

- ** Density ** - mass per unit volume of a substance
- ** Specific Volume ** - volume occupied by a unit mass of a substance; inverse of density

Densities of pure solids and liquids are essentially independent of pressure and vary only slightly with temperature. Densities of some pure compounds, solutions, and mixtures can be found in standard references

Density can be used as a conversion factor to relate mass and volume of a quantity of the substance.

In [10]:
import pint
ureg = pint.UnitRegistry()

V = 20.0 * ureg.centimeter**3
rho = 1.595 * ureg.gram / ureg.centimeter**3

print((V * rho).dimensionality == '[mass]')
m = V * rho

print(m)

True
31.9 gram


In [18]:
# import pint
# ureg = pint.UnitRegistry()

m = 6.20 * ureg.pound
m.ito(ureg.gram)
rho = 1.595 * ureg.gram / (1 * ureg.centimeter**3)

print((m / rho).dimensionality == '[length] ** 3')
V = m / rho
print(V)

True
1763.1803724137933 centimeter ** 3


** Specific gravity ** - ratio of the density, $\rho$, of he substance to the density, $\rho_{ref}$, of a reference substance at a specified condition

$$ \text{SG} = \rho/\rho_{ref} $$

Most commonly use water at 4 degrees Celsius

$$ \rho_{\text{H}_2\text{O(l)}} = 
1.000 \text{g/cm}^3 =
1000. \text{kg/m}^3 =
2.43 \text{lb}_m\text{/ft}^3
$$

Density of a liquid or solid in $\text{g/cm}^3$ is numerically equal to the specific gravity of that substance. Below:

$$ \text{SG} = 0.6\frac{20\text{ degrees}}{4\text{ degrees}} $$

signifies the specific gravity of a substance at 20 degrees C with reference to water at 4 degrees C is 0.6.

*If you are given the SG of a substance, multiply it by the reference density in any units to get the density of the substance in the same units*

- ** Note ** - Some special densities such as degrees API are occasionally used (especially in petroleum industry). See references like Perry's Handbook for definitions and conversion factors

### Test Youself

#### Problem 1
What are the units of specific gravity?
#### Solution 1
It does not have any units. It is the substance density / reference density

#### Problem 2
A liquid has a specific gravity of 0.50:
- What is its density in $\text{g/cm}^3$? 
- What is its specific volume in $\text{cm}^3\text{/g}$? 
- What is its density in $\text{lb}_m \text{/ft}^3$?
- What is the mass of 3.0 $\text{cm}^3$ of this liquid?
- What volume is occupied by 18 g?

In [3]:
# calcs for Problem 2
# import pint
# ureg = pint.UnitRegistry()

print(0.5*3)
print((0.50 * ureg.gram/ureg.centimeter**3).to(ureg.pound/ureg.foot**3))
print(2*18)

1.5
31.213980288072296 pound / foot ** 3
36


#### Solution 2
- $0.50 \text{g/cm}^3$
- $2.0  \text{cm}^3/\text{g}$
- Convert $0.50 \text{g/cm}^3$ to $\text{lb}_m/\text{ft}^3$ = $ 31\text{lb}_m/\text{ft}^3$
- $0.50 \times 3.0 = 1.5 \text{g}$
- $2.0 \times 18 = 36 \text{cm}^3$

#### Problem 3
If substance A and substance B each have a density of 1.34 $\text{g/cm}^3$, must 3 $\text{cm}^3$ of A have the same mass as 3 $\text{cm}^3$ of B?
#### Solution 3
Yes

#### Problem 4
If substance A and substance B each have a specific gravity of 1.34, must 3 $\text{cm}^3$ of A have the same mass as 3 $\text{cm}^3$ of B?
#### Solution 4
Yes only if same reference density is used. No otherwise.

#### Problem 5
Freezing a sealed full bottle of water leads to a broken bottle and freezing a sealed full flexible-walled contained of n-butyl alcohol leads to a container with concave walls. What can you conclude about the densities of the solid and liquid forms of these two susbtances?
#### Solution 5
Water's density decreases (expands) as it freezes, n-butyl alcohol's density increases as it freezes (contracts)

#### Problem 6
Does the density of liquid mercury increase or decrease with increasing temperature? Justify your answer using a thermometer as an illustration
#### Solution 6
The density of liquid mercury decreases as it increases in temperature. This causes it to expand and move up the height of a thermometer.

### Example 3.1-1 Mass, Volume, and Density

Calculate the density of mercury in $\text{lb}_m/\text{ft}^3$ from a tabulated specific gravity, and calculate the volume in $\text{ft}^3$ occupied by $215\text{ kg}$ of mercury

#### Solution

$$ 
\text{SG} = 13.546 \
$$

$$ 
\rho_{Hg} = 
(13.546)(62.43\frac{\text{lb}_m}{\text{ft}^3}) = 845.7\frac{\text{lb}_m}{\text{ft}^3} 
$$

$$ 
V = 215 \text{ kg} \times 
\frac{1 \text{ lb}_m}{0.454 \text{ kg}} 
\times \frac{1 \text{ ft}^3}{845.7 \text{ lb}_m} =
0.560 \text{ ft}^3
$$

From *Perry's Chemical Engineers' Handbook*, the dependence volume of mercury on temperature is:

$$
V(T) = V_0(1 + 0.18182 \times 10^{-3}T + 0.0078 \times 10^{-6}T^2)
$$

where $V(T)$ is the volume of a given mass of mercury at temperature $T(C)$ and $V_0$ is the volume of the same mass of mercury at $0C$.

### Example 3.1-2 Effect of Temperature on Liquid Density

In Example 3.1-1, 215 kg of mercury was found to occupy 0.560 $\text{ft}^3$ at 20 degrees Celsius. 
- What volume would the mercury occupy at 100 degrees Celsius?
- Suppose the mercury is contained in a cylinder having a diameter of 0.25 in. What change in height would be observed as the mercury is heated from 20 degrees Celsius to 100 degrees Celsius?

In [10]:
from sympy import *
V, T, V0= symbols('V T V0')

# create equation
V = V0 * (1 + 0.18182 * 10**-3 * T + 0.0078 * 10**-6 * T**2)
print(V)

V0*(7.8e-9*T**2 + 0.00018182*T + 1)


In [13]:
# Solve for V0 using given info for 20 deg C
solve(Eq(0.560,V.subs(T, 20)), V0)

[0.557969259719864]

In [16]:
# Use V0 in V(T=100)
V.subs([(T,100),(V0,0.557969)])

0.568157513940000

#### Solution 1
$$V(100) = 0.568 \text{ft}^3$$

#### Solution 2

Volume of Hg equals $\pi D^2H/4$ where $D$ is the cylinder diamter and $H$ is its height. Since $D$ is constant,

$$
H(100) - H(20) = \frac{V(100) - V(20)}{\pi D^2/ 4}
$$

$$
D = (0.25/12)\text{ft}
$$

$$ = 23.5 \text{ft} $$

In [27]:
# from sympy import *
V, T = symbols('V T')

# use previously calc'd V0 and convert D to feet
V0 = 0.557969259719864
D = 0.25/12

# create equation
V = V0 * (1 + 0.18182 * 10**-3 * T + 0.0078 * 10**-6 * T**2)
print(V)

4.35216022581494e-9*T**2 + 0.000101449970802266*T + 0.557969259719864


In [34]:
# calc delta H; use pi.evalf() to get numerical result for pi
H = (V.subs(T,100) - V.subs(T,20))/(pi.evalf() * D**2 / 4)
print(H)

23.9312011600681
